cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -41,8 +41,12 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 // CNV-4078 UI pause/unpause
 
-// CNV-4310/CNV-3881 SVVP
-
+// CNV-3881 Release note to announce SVVP Certification
+* {CNVProductNameStart} is certified in Microsoft's Windows Server
+Virtualization Validation Program (SVVP) to run Windows Server workloads.
+** SVVP Certification applies to Intel and AMD CPUs.
+** SVVP Certificate RHCOS Core OS8 workers are named _Red Hat OpenShift
```

Comment:
According to the master product sheet:
"RHCOS Core OS8" should be "Red Hat Enterprise Linux CoreOS 8" at first usage

Reply:
Cool, cool. Nice catch!